lib/src/services/ds_bottom_sheet.service.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSBottomSheetService {
+  final BuildContext context;
+  final Widget child;
+
+  DSBottomSheetService({
+    required this.context,
+    required this.child,
+  });
+
+  Widget _buildBottomSheet({ScrollController? controller}) {
+    return Container(
+      decoration: const BoxDecoration(
+        color: DSColors.neutralLightSnow,
+        borderRadius: BorderRadius.only(
+          topLeft: Radius.circular(15.0),
+          topRight: Radius.circular(15.0),
+        ),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.symmetric(
+          horizontal: 16.0,
+          vertical: 8.0,
+        ),
+        child: Stack(
+          children: [
+            Row(
+              mainAxisSize: MainAxisSize.max,
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Container(
+                  height: 4.0,
+                  width: 32.0,
+                  decoration: const BoxDecoration(
+                    color: DSColors.neutralMediumWave,
+                    borderRadius: BorderRadius.all(
+                      Radius.circular(15.0),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+            Padding(
+              padding: const EdgeInsets.only(top: 8.0),
```

Comment:
This padding should consider the system navigation bar.

<img width="886" alt="image" src="https://user-images.githubusercontent.com/91232521/195172935-ad312371-4939-4fdf-bea7-ce8859d9e724.png">

```dart
padding: EdgeInsets.only(
  top: 8.0,
  bottom: MediaQueryData.fromWindow(window).padding.bottom + 36,
),
```

Reply:
done